OWNERS

```diff
@@ -8,3 +8,7 @@ reviewers:
   - AlonaKaplan
   - eranco74
   - tsorya
+
+# Bugzilla info;
+component: Installer
```

Comment:
This is the last place I would be opening bug on regarding kubernetes-nmstate day-2 network configuration... Can't we get a new subcomponent under Networking instead?

Reply:
@phoracek , Well, I just copy-paste BZ details from one of our bugs.
I guess we can change it anytime, if you have another suggestion - I'm open.

Reply:
@celebdor could we get a new subcomponent under Networking? It would be clearer where to open bugs for users.

Reply:
yes. I'll request it now

Reply:
In the meantime, we should probably use Networking, not Installer